dev/source/docs/ros2-gazebo.rst

```diff
@@ -17,16 +17,28 @@ Ensure you have the prerequisites complete and working before beginning this Gaz
 Install Gazebo
 ==============
 
-First, install `Gazebo Harmonic (recommended) <https://gazebosim.org/docs/harmonic/install>`__ or `Gazebo Garden <https://gazebosim.org/docs/garden/install>`__.
+First, install `Gazebo Harmonic (recommended) <https://gazebosim.org/docs/harmonic/install>`__ or `Gazebo Garden <https://gazebosim.org/docs/garden/install>`__. Please check the `Compatibility Matrix between ROS and Gazebo versions <https://gazebosim.org/docs/latest/ros_installation/#summary-of-compatible-ros-and-gazebo-combinations>`__.
 
 Next, set up all the necessary ROS 2 packages in the workspace.
 
-We will clone the required repositories using `vcstool <https://github.com/dirk-thomas/vcstool>`__ and a `ros2.repos` files:
+We will clone the required repositories using `vcstool <https://github.com/dirk-thomas/vcstool>`__ and a `.repos` files:
 
-.. code-block:: bash
+.. tabs::
+
+    .. tab:: Jazzy
+
+        .. code-block:: bash
+
+            cd ~/ardu_ws
+            vcs import --input https://raw.githubusercontent.com/ArduPilot/ardupilot_gz/main/jazzy_gz.repos --recursive src
+            
+    .. tab:: Humble
+
+        .. code-block:: bash
+
+            cd ~/ardu_ws
+            vcs import --input https://raw.githubusercontent.com/ArduPilot/ardupilot_gz/main/humble_gz.repos --recursive src
```

Comment:
for `ardupilot_gz`, we could separate branches `Jazzy` and `Humble`, but since most of the development team is still on `Humble` that might cause headaches syncing the two branches. Simply having two `.repos` files is proposed instead.

Reply:
I am happy with either approach. In a few other repos in Github, I started using mergify.io, so the work to maintain multiple branches is very low.